express-crud/server.js

```diff
@@ -0,0 +1,143 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib';
+
+const app = express();
+
+app.use(express.json());
+
+const db = new pg.Pool();
```

Comment:
How does this work? There is not anything that tells Postgres which database to use.